src/core/layers/survey.py

```diff
@@ -0,0 +1,73 @@
+# coding: utf8
+
+__contact__ = "info@hytech-imaging.fr"
+__copyright__ = "Copyright (c) 2021 Hytech Imaging"
+
+from qgis.core import QgsEditorWidgetSetup, QgsVectorLayer
+from ..database import (
+    SammoDataBase,
+    SURVEY_TABLE,
```

Comment:
import order

Reply:
I changed it, even if github doesn't say the comment is outdated :thinking: